src/pages/Home/index.js

```diff
@@ -1,21 +1,21 @@
 import Button from 'components/common/Button';
+import MapView from 'components/mapView';
+import useGeoLocation from 'hooks/useGeolocation';
 import useTranslation from 'hooks/useTranslation';
 import { useLogoutMutation } from 'services/auth/auth';
 
-import logo from 'assets/logo.svg';
-
-import './styles.css';
+import './styles.scss';
 
 const Home = () => {
   const t = useTranslation();
   const [logout, { isLoading }] = useLogoutMutation();
 
   const handleLogout = () => logout().then(() => localStorage.removeItem('user'));
+  const location = useGeoLocation();
 
   return (
     <div className="home">
-      <img src={logo} className="home__logo" alt={t('home.logoAltMsg')} />
-      <h1>{t('home.welcomeMsg')}</h1>
+      <MapView location={location} />
```

Comment:
Since you created a hook for the geolocation, maybe you can use it directly on the MapView component instead of passing it as a parameter.